From tklauser@distanz.ch  Tue Apr 28 15:57:46 2009
From: Tobias Klauser <tklauser@distanz.ch>
Date: Sun, 26 Apr 2009 15:06:09 +0200
Subject: Staging: pohmelfs: Storage class should be before const qualifier
To: gregkh@suse.de, zbr@ioremap.net
Cc: Tobias Klauser <tklauser@distanz.ch>
Message-ID: <1240751169-10167-1-git-send-email-tklauser@distanz.ch>


The C99 specification states in section 6.11.5:

The placement of a storage-class specifier other than at the beginning
of the declaration specifiers in a declaration is an obsolescent
feature.

Signed-off-by: Tobias Klauser <tklauser@distanz.ch>
Acked-by: Evgeniy Polyakov <zbr@ioremap.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/pohmelfs/inode.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/pohmelfs/inode.c
+++ b/drivers/staging/pohmelfs/inode.c
@@ -943,7 +943,7 @@ err_out_unlock:
 	return ret;
 }
 
-const static struct file_operations pohmelfs_file_ops = {
+static const struct file_operations pohmelfs_file_ops = {
 	.open		= generic_file_open,
 	.fsync		= pohmelfs_fsync,
 
